### PR TITLE
need to fix BE-api

### PR DESCRIPTION
**Pull request by mini-wizbit**

### CODE REVIEW

**Comment by mini-wizbit** [comment]
YAY fixed the back end stuff and now works

File: nc-games/src/utils/api.js [hunk 1]
```diff
@@ -4,8 +4,24 @@ const gamesApi = axios.create({
...
+
+export const getCategories = () => {
+  return gamesApi.get("/categories").then(({ data }) => {
+    return data.categories;
+  });
+};
+
+export const getReviewsByCategories = (category) => {
+  return gamesApi.get(`/reviews?category=${category}`).then(({ data }) => {
+    return data.reviews;
```
Comment (hevdevs) [inline nc-games/src/utils/api.js:25]: to make this (getReviewsByCategory) DRYer, you could refactor to make category an optional parameter for getReviews and then check if category is defined or not before making your get request

File: nc-games/src/utils/api.js [hunk 2]
```diff
@@ -4,8 +4,24 @@ const gamesApi = axios.create({
...
 });
 
-export const getReviews = () => {
-  return gamesApi.get(`/reviews`).then(({ data }) => {
+export const getReviews = (category) => {
+  let queryStr = "/reviews";
+  if (category) queryStr += `?category=${category}`;
+  console.log(queryStr, "< is this right?");
+  return gamesApi.get(queryStr).then(({ data }) => {
+    console.log(data, "from api");
```
Comment (hevdevs) [inline nc-games/src/utils/api.js:12]: remove console.logs

**Review by hevdevs — APPROVED** [review]
couple little nitpicks - feel free to merge after removing console logs 🎉

**Pull request merged**